tslint.json

```diff
@@ -77,6 +77,18 @@
       "check-format",
       "allow-leading-underscore",
       "allow-pascal-case"
+    ],
+    "whitespace": [true,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        // "check-module", <- this rule is not compatible with
```

Comment:
> Newline is newline. Whitespace is whitespace.

I don't think it's so black and white. For the purposes of the whitespace rule where you want to see a space after `{` and before `}` in a single-line import statement, the rule _should_ consider newlines in multiline import statements as whitespace. I still expect the following to produce failures:

```ts
import {a,
        ~
  b} from "foo";
   ~
```

but @andy-hanson's example should be valid. 

Being pedantic about the semantics of what a "whitespace" is vs. what a "newline" is doesn't help us here; what matters most is the intent of the lint rule and the formatting or style consistency it helps us achieve.